src/app/complex/complex-results/complex-results.module.ts

```diff
@@ -27,7 +27,7 @@ import {
 import {
   TableSubcomponentInteractorComponent
 } from './complex-navigator/table-structure/table-interactor-column/table-subcomponent-interactor/table-subcomponent-interactor.component';
-import {ComplexNavigatorSortingComponent} from './complex-navigator-sorting/complex-navigator-sorting.component';
+import {ComplexNavigatorSortingComponent} from './complex-navigator-buttons/complex-navigator-sorting.component';
```

Comment:
You rename the component selector `cp-complex-navigator-buttons` and rename the folder containing the component `complex-navigator-buttons`; but the actual component remains with the old name.

Files inside ``complex-navigator-buttons`:
- complex-navigator-sorting.component.ts
- complex-navigator-sorting.component.spec.ts
- complex-navigator-sorting.component.css
- complex-navigator-sorting.component.html

And the component is still called `ComplexNavigatorSortingComponent`.

Nothing important but we should rename all of this for consistency.